src/NUnitFramework/framework/Constraints/CollectionItemsEqualConstraint.cs

```diff
@@ -37,7 +37,7 @@ public abstract class CollectionItemsEqualConstraint : CollectionConstraint
         /// <summary>
         /// The NUnitEqualityComparer in use for this constraint
         /// </summary>
-        private readonly NUnitEqualityComparer _comparer = new NUnitEqualityComparer();
+        protected internal NUnitEqualityComparer Comparer { get; } = new NUnitEqualityComparer();
```

Comment:
In order to wrap this in an `IComparer` implementation in a child class (UniqueItemsConstraint)

Reply:
Can this be `private protected`?

Reply:
Oh, sorry. I just checked MSDN. I had thought this would available to types that were both derived and within the assembly (rather than either). I'll update this next chance I get.